modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -804,18 +835,24 @@ def validate_account_values!
         account_number = direct_deposit_account_vals&.dig('accountNumber')
         routing_number = direct_deposit_account_vals&.dig('routingNumber')
 
-        form_object_desc = 'direct deposit'
-
         if account_type.blank? || valid_account_types.exclude?(account_type)
-          raise_exception_if_value_not_present('account type (CHECKING/SAVINGS)', form_object_desc)
+          collect_error_messages(detail: 'accountType is missing or blank',
```

Comment:
Maybe we should capitalize each one of these initial words in each sentence, "The account type is missing...", "The account number...", and "The routing number...".